assets/xml/objects/object_rb.xml

```diff
@@ -1,44 +1,44 @@
+        <DList Name="gLeeverBottomHalfWrapperDlist2" Offset="0x2D0" />
+        <DList Name="gLeeverBottomHalfWrapperDlist1" Offset="0x2E0" />
+        <DList Name="gLeeverTopHalfDlist" Offset="0x430" />
+        <DList Name="gLeeverTopHalfWrapperDlist2" Offset="0x570" />
+        <DList Name="gLeeverTopHalfWrapperDlist1" Offset="0x580" />
+        <DList Name="gLeeverSpike1Dlist" Offset="0x760" />
+        <DList Name="gLeeverSpike1WrapperDlist2" Offset="0x7F8" />
+        <DList Name="gLeeverSpike1WrapperDlist1" Offset="0x808" />
+        <DList Name="gLeeverSpike2Dlist" Offset="0x8B8" />
+        <DList Name="gLeeverSpike2WrapperDlist2" Offset="0x950" />
+        <DList Name="gLeeverSpike2WrapperDlist1" Offset="0x960" />
+        <DList Name="gLeeverSpike3Dlist" Offset="0xA10" />
+        <DList Name="gLeeverSpike3WrapperDlist2" Offset="0xAB0" />
+        <DList Name="gLeeverSpike3WrapperDlist1" Offset="0xAC0" />
+        <DList Name="gLeeverSpike4Dlist" Offset="0xB80" />
+        <DList Name="gLeeverSpike4WrapperDlist2" Offset="0xC18" />
+        <DList Name="gLeeverSpike4WrapperDlist1" Offset="0xC28" />
+        <Texture Name="gLeeverSpikeTex" OutName="spike" Format="rgba16" Width="8" Height="16" Offset="0xCD8" />
+        <Texture Name="gLeeverSideTex" OutName="side" Format="rgba16" Width="32" Height="32" Offset="0xDD8" />
+        <Texture Name="gLeeverFlowerTex" OutName="flower" Format="rgba16" Width="32" Height="32" Offset="0x15D8" />
+        <Limb Name="object_rb_Standardlimb_001DD8" Type="Standard" Offset="0x1DD8" />
+        <Limb Name="object_rb_Standardlimb_001DE4" Type="Standard" Offset="0x1DE4" />
+        <Limb Name="object_rb_Standardlimb_001DF0" Type="Standard" Offset="0x1DF0" />
+        <Limb Name="gLeeverTopHalfLimb" Type="Standard" Offset="0x1DFC" />
+        <Limb Name="object_rb_Standardlimb_001E08" Type="Standard" Offset="0x1E08" />
+        <Limb Name="object_rb_Standardlimb_001E14" Type="Standard" Offset="0x1E14" />
+        <Limb Name="gLeeverSpike1Limb" Type="Standard" Offset="0x1E20" />
+        <Limb Name="object_rb_Standardlimb_001E2C" Type="Standard" Offset="0x1E2C" />
+        <Limb Name="object_rb_Standardlimb_001E38" Type="Standard" Offset="0x1E38" />
+        <Limb Name="gLeeverSpike2Limb" Type="Standard" Offset="0x1E44" />
+        <Limb Name="object_rb_Standardlimb_001E50" Type="Standard" Offset="0x1E50" />
+        <Limb Name="object_rb_Standardlimb_001E5C" Type="Standard" Offset="0x1E5C" />
+        <Limb Name="gLeeverSpike3Limb" Type="Standard" Offset="0x1E68" />
+        <Limb Name="object_rb_Standardlimb_001E74" Type="Standard" Offset="0x1E74" />
+        <Limb Name="object_rb_Standardlimb_001E80" Type="Standard" Offset="0x1E80" />
+        <Limb Name="gLeeverSpike4Limb" Type="Standard" Offset="0x1E8C" />
+        <Limb Name="gLeeverBottomHalfLimb" Type="Standard" Offset="0x1E98" />
+        <Skeleton Name="gLeeverSkel" Type="Normal" LimbType="Standard" Offset="0x1EE8" />
```

Comment:
Try to not remove the placeholders `EnumName`s, and possibly name the ones you named
```suggestion
        <Limb Name="object_rb_Standardlimb_001DD8" Type="Standard" EnumName="OBJECT_RB_LIMB_01" Offset="0x1DD8" />
        <Limb Name="object_rb_Standardlimb_001DE4" Type="Standard" EnumName="OBJECT_RB_LIMB_02" Offset="0x1DE4" />
        <Limb Name="object_rb_Standardlimb_001DF0" Type="Standard" EnumName="OBJECT_RB_LIMB_03" Offset="0x1DF0" />
        <Limb Name="gLeeverTopHalfLimb" Type="Standard" EnumName="LEEVER_LIMB_TOP_HALG" Offset="0x1DFC" />
        <Limb Name="object_rb_Standardlimb_001E08" Type="Standard" EnumName="OBJECT_RB_LIMB_05" Offset="0x1E08" />
        <Limb Name="object_rb_Standardlimb_001E14" Type="Standard" EnumName="OBJECT_RB_LIMB_06" Offset="0x1E14" />
        <Limb Name="gLeeverSpike1Limb" Type="Standard" EnumName="LEEVER_LIMB_SPIKE1" Offset="0x1E20" />
        <Limb Name="object_rb_Standardlimb_001E2C" Type="Standard" EnumName="OBJECT_RB_LIMB_08" Offset="0x1E2C" />
        <Limb Name="object_rb_Standardlimb_001E38" Type="Standard" EnumName="OBJECT_RB_LIMB_09" Offset="0x1E38" />
        <Limb Name="gLeeverSpike2Limb" Type="Standard" EnumName="LEEVER_SPIKE_2" Offset="0x1E44" />
        <Limb Name="object_rb_Standardlimb_001E50" Type="Standard" EnumName="OBJECT_RB_LIMB_0B" Offset="0x1E50" />
        <Limb Name="object_rb_Standardlimb_001E5C" Type="Standard" EnumName="OBJECT_RB_LIMB_0C" Offset="0x1E5C" />
        <Limb Name="gLeeverSpike3Limb" Type="Standard" EnumName="LEEVER_LIMB_SPIKE_3" Offset="0x1E68" />
        <Limb Name="object_rb_Standardlimb_001E74" Type="Standard" EnumName="OBJECT_RB_LIMB_0E" Offset="0x1E74" />
        <Limb Name="object_rb_Standardlimb_001E80" Type="Standard" EnumName="OBJECT_RB_LIMB_0F" Offset="0x1E80" />
        <Limb Name="gLeeverSpike4Limb" Type="Standard" EnumName="LEEVER_LIMB_SPIKE_4" Offset="0x1E8C" />
        <Limb Name="gLeeverBottomHalfLimb" Type="Standard" EnumName="LEEVER_LIMB_BOTTOM_HALF" Offset="0x1E98" />
        <Skeleton Name="gLeeverSkel" Type="Normal" LimbType="Standard" LimbNone="LEEVER_LIMB_NONE" LimbMax="LEEVER_LIMB_MAX" EnumName="LeeverLimbs" Offset="0x1EE8" />
```

Reply:
oh, i think what happened there is i pulled master, it probably added those and i didn't notice, so i just nuked that part of the conflict